troposphere/static/css/app/quota.scss

```diff
@@ -0,0 +1,40 @@
+textarea, select{
+	width: 95%;
+}
+
+.quota-table{
+	float: left;
+	width: 50%;
+}
+
+.quota-detail{
+	width: 48%;
+	float: right;
+	position: relative;
+	border: 5px solid #007b89;
+}
+
+.quota-detail button{
+	margin: 2%;
+}
+
+.quota-detail div{
+	margin: 2%;
+}
+
+.quota-row, .quota-inline{
+	width: 100%;
+}
+
+.user-name a.active{
+	color: #007b89;
```

Comment:
It was meant to be the same color as the navbar background but I realize now that I could use the $navbar-default-bg SCSS variable.